docs/user-guide/rules.md

```diff
@@ -360,6 +360,7 @@ Enforce one representation of things that have multiple with these `notation` (s
 | [`keyframe-selector-notation`](../../lib/rules/keyframe-selector-notation/README.md)<br/>Specify keyword or percentage notation for keyframe selectors. | ✅ | 🔧 |
 | [`lightness-notation`](../../lib/rules/lightness-notation/README.md)<br/>Specify number or percentage notation for lightness. | ✅ | 🔧 |
 | [`media-feature-range-notation`](../../lib/rules/media-feature-range-notation/README.md)<br/>Specify context or prefix notation for media feature ranges. | ✅ | 🔧 |
+| [`selector-nesting-notation`](../../lib/rules/selector-nesting-notation/README.md)<br/>Specify explicit or implicit notation for nesting selectors. | ✅ | 🔧 |
```

Comment:
Putting it in standard would be very opinionated.
It shouldn't be part of it.

Reply:
Not everything is set in stone by the comments of the issue.
Some things change while reviewing the PR.

Reply:
> What do you mean?

Transition from a CSS preprocessor to CSS nesting.